apps/elewa-group-website/src/app/app.routing.ts

```diff
@@ -0,0 +1,28 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Route }  from '@angular/router';
+
+export const ELEWA_GROUP_WEBSITE_ROUTES: Route[] = [
+
+  // App Entry-Point
+
+  { path: '', redirectTo: `home`, pathMatch: 'full' },
+  {  path: 'home', redirectTo: 'home/en'  },
+
```

Comment:
This route needs to go to the home module and have a default path to the HomePageComponent.

However, routing was not a concern of this issue